amr-wind/overset/overset_ops_routines.H

```diff
@@ -0,0 +1,499 @@
+#ifndef OVERSET_OPS_ROUTINES_H_
+#define OVERSET_OPS_ROUTINES_H_
+
+#include "amr-wind/equation_systems/vof/volume_fractions.H"
+#include "amr-wind/overset/overset_ops_K.H"
+
+namespace amr_wind::overset_ops {
+
+// Populate approximate signed distance function using vof field
+void populate_psi(
+    amrex::MultiFab& mf_psi,
+    amrex::MultiFab& mf_vof,
+    const amrex::Real i_th,
+    const amrex::Real asdf_tiny)
+{
+    for (amrex::MFIter mfi(mf_psi); mfi.isValid(); ++mfi) {
```

Comment:
for something like this, lets do the fused mfiter stuff. It will avoid later conversions, automatically will work with openmp, etc. There are bunch of these in this file.

Reply:
I converted all but two of the loops, which iterate on faces in each coordinate direction. let me know if you think changing those would be worth it.

Reply:
Maybe? My thinking is that each loop that remains unfused that could be is a potential opportunity for performance gains and also avoids more copy/paste of inefficient stuff. I could help out if need be too. Thank you for doing this.

Reply:
if I supply a nodal mfab to the ParallelFor, it will go over a nodal box, right?

Reply:
Yeah I would expect it to... but probably good to double check.

Reply:
done